src/IO/S3Common.cpp

```diff
@@ -40,26 +40,45 @@ const std::pair<DB::LogsLevel, Poco::Message::Priority> & convertLogLevel(Aws::U
 class AWSLogger final : public Aws::Utils::Logging::LogSystemInterface
 {
 public:
+    AWSLogger()
+    {
+        log = &Poco::Logger::get("AWSClient");
+        tag_loggers["AWSAuthV4Signer"] = &Poco::Logger::get("AWSClient (AWSAuthV4Signer)");
```

Comment:
Where are these string tokens for tags defined/generated? May be they can be imported? I found something in contrib/aws/aws-cpp-sdk-core/source/auth/AWSAuthSigner.cpp:67:
`static const char v4LogTag[] = "AWSAuthV4Signer";`

Reply:
Cool, good catch!

Reply:
But they are not exposed in any header :(

Reply:
I suggest, that we declare these tags in S3Common the same way as they are declared in `contrib/aws`. This can be done in a separate PR, though.

Reply:
+